modules/serverless-kafka-source-odc.adoc

```diff
@@ -24,7 +24,7 @@ After the Knative broker implementation for Apache Kafka is installed on your cl
 .. Add a comma-separated list of *Topics*.
 .. Add a *Consumer Group*.
 .. Select the *Service Account Name* for the service account that you created.
-.. Select the *Sink* for the event source. A *Sink* can be either a *Resource*, such as a channel, broker, or service, or a *URI*.
+.. Select the *Target* for the event source. A *Sink* can be either a *Resource*, such as a channel, broker, or service, or a *URI*.
```

Comment:
Just double-checking that *Target* in the first sentence and *Sink* in the second sentence are both intended to coexist.